betelgeuse.py

```diff
@@ -574,6 +621,40 @@ def cli(context, jobs):
     testimony.SETTINGS['minimum_tokens'] = ['id']
 
 
+@cli.command('create-plan')
+@click.option(
+    '--name',
+    default='test-plan-{0}'.format(time.time()),
+    help='Name for new Test Plan.',
+)
+@click.option(
+    '--type',
+    default='release',
+    help='Test Plans can be Releases or Iterations.',
+    type=click.Choice([
+        'release',
+        'iteration',
+    ])
+)
+@click.option(
+    '--parent-name',
+    help='Name of parent Test Plan to link to.',
+)
+@click.option(
+    '--collect-only',
```

Comment:
I am not sure about the collect only here. I think it can be dropped because just printing that a test-plan will be created is something already expected by the command.

Collect only is useful for the test-case command because the user can simulate all the test cases and requirements that should be created and that produces a huge amount of output depending on the code base.